Resources/Prototypes/Alerts/alerts.yml

```diff
@@ -69,22 +69,19 @@
   icon: /Textures/Interface/Alerts/Weightless/weightless.png
   name: Weightless
   description: >-
-    Gravity has ceased affecting you, and you're floating around aimlessly. You'll need something large and heavy, like a
-    wall or lattice, to push yourself off if you want to move. A jetpack would enable free range of motion. A pair of
-    magboots would let you walk around normally on the floor. Barring those, you can throw things, use a fire extinguisher,
-    or shoot a gun to move around via Newton's 3rd Law of Motion.
+    Gravity has ceased affecting you, and you're floating around aimlessly. Find something sturdy to hold onto, or expand force in a direction opposite of you. Magnetic boots or jetpacks will help you move with more control
 
 - type: alert
   alertType: Stun
-  icon: /Textures/Objects/Weapons/Melee/stunbaton.rsi/stunbaton_off.png
+  icon: /Textures/Interface/Alerts/Stun/Stunned.png #Should probably add a proper Icon
```

Comment:
I'm assuming you copy/pasted the stunbaton into the Alerts folder. Don't do that, it was fine before.